### ISSUE TITLE
vars-on-top module issue

### ISSUE THREAD

**Issue by jonathanKingston** [babel/babel-eslint#49]
I see the following error:
```
app.js
  8:0  error  All "var" declarations must be at the top of the function scope  vars-on-top
```

With this code:
```
import Ember from 'ember';
import Resolver from 'ember/resolver';
import loadInitializers from 'ember/load-initializers';
import config from './config/environment';

Ember.MODEL_FACTORY_INJECTIONS = true;

var App = Ember.Application.extend({
  modulePrefix: config.modulePrefix,
  podModulePrefix: config.podModulePrefix,
  Resolver: Resolver
});

loadInitializers(App, config.modulePrefix);

export default App;
```

This silences the error however certainly looks wrong:
```
var App;
import Ember from 'ember';
import Resolver from 'ember/resolver';
import loadInitializers from 'ember/load-initializers';
import config from './config/environment';

Ember.MODEL_FACTORY_INJECTIONS = true;

App = Ember.Application.extend({
  modulePrefix: config.modulePrefix,
  podModulePrefix: config.podModulePrefix,
  Resolver: Resolver
});

loadInitializers(App, config.modulePrefix);

export default App;
```

I'm not really sure if this should be raised here or part of ESLint itself or if it will just resolve itself when ESLint supports import natively soon, however I just thought I would track the progress here.

**Comment by srigi** [comment]
This is addressed in [eslint itself](https://github.com/eslint/eslint/issues/2099)

**Comment by jonathanKingston** [comment]
@srigi :+1: thanks for addressing there, I had not had time to diagnose properly.
